utils/mediaupload.js

```diff
@@ -49,8 +49,19 @@ export function mediaUpload( {
 
 	// Allowed types for the current WP_User
 	const allowedMimeTypesForUser = get( window, [ '_wpMediaSettings', 'allowedMimeTypes' ] );
+	// Browsers may use unexpected mime types and they defer from browser to browser.
+	// Here we compute additional possible mime types.
+	// This allows us to be more flexible we don't want to block files that the server would accept.
+	// Converts { jpg|jpeg|jpe: "image/jpeg" } into [ "image/jpeg", "image/jpg", "image/jpeg", "image/jpe" ]
+	// This solves the problem in chrome where mp3 files have audio/mp3 instead of audio/mpeg.
+	// https://bugs.chromium.org/p/chromium/issues/detail?id=227004
+	const flexibleAllowedMimeTypes = flatMap( allowedMimeTypesForUser, ( mime, extensionsString ) => {
+		const [ generalMimeType ] = mime.split( '/' );
```

Comment:
Minor: Canonical name for this value is its "type" (the part after slash being its "subtype"):

https://en.wikipedia.org/wiki/Media_type#Naming